src/sensor/ping.h

```diff
@@ -321,6 +329,11 @@ class Ping : public PingSensor
     Q_INVOKABLE void firmwareUpdate(QString fileUrl, bool sendPingGotoBootloader = true, int baud = 57600,
                                     bool verify = true);
 
+    /**
+     * @brief request a profile message from the device
+     */
+    Q_INVOKABLE void emitPing() { request(Ping1dId::PROFILE); }
```

Comment:
const function.

Reply:
done, and now Sensor::request is const too as a requirement.